doc/sphinx/changes.rst

```diff
@@ -8,6 +8,675 @@ Recent changes
+Semeria, Michael Soegtrop, Arnaud Spiwack, Matthieu Sozeau, Enrico
+Tassi, Laurent Théry, Anton Trunov, Li-yao Xia and Théo Zimmermann
+
+The 52 contributors to this version are Reynald Affeldt, Tanaka Akira, Frédéric
+Besson, Lasse Blaauwbroek, Clément Blaudeau, Martin Bodin, Ali Caglayan, Tej Chajed,
+Cyril Cohen, Julien Coolen, Matthew Dempsky, Maxime Dénès, Andres Erbsen,
+Jim Fehrle, Emilio Jesús Gallego Arias, Paolo G. Giarrusso, Attila Gáspár, Gaëtan Gilbert,
+Jason Gross, Benjamin Grégoire, Hugo Herbelin, Wolf Honore, Jasper Hugunin, Ignat Insarov,
+Ralf Jung, Fabian Kunze, Vincent Laporte, Olivier Laurent, Larry D. Lee Jr,
+Thomas Letan, Yishuai Li, Xia Li-yao, James Lottes, Jean-Christophe Léchenet,
+Kenji Maillard, Erik Martin-Dorel, Yusuke Matsushita, Guillaume Melquiond,
+Carl Patenaude-Poulin, Clément Pit-Claudel, Pierre-Marie Pédrot, Pierre Roux,
+Kazuhiko Sakaguchi, Vincent Semeria, Michael Soegtrop, Matthieu Sozeau,
+Enrico Tassi, Anton Trunov, Edward Wang, Li-yao Xia, Beta Ziliani and Théo Zimmermann.
+
+The Coq community at large helped improve the design of this new version via
+the GitHub issue and pull request system, the Coq development mailing list
+coqdev@inria.fr, the coq-club@inria.fr mailing list, the `Discourse forum
+<https://coq.discourse.group/>`_ and the `Coq Zulip chat <http://coq.zulipchat.com>`_.
+
+Version 8.13's development spanned 5 months from the release of
+Coq 8.12.0. Enrico Tassi and Maxime Dénès are the release managers of Coq 8.13.
+This release is the result of 400 merged PRs, closing ~100 issues.
+
+| Nantes, November 2020,
+| Matthieu Sozeau for the Coq development team
+|
+
+
+Changes in 8.13
+~~~~~~~~~~~~~~~
+
+.. contents::
+   :local:
+
+Kernel
+^^^^^^
+
+- **Added:**
+  Definitional UIP, only when :flag:`Definitional UIP` is enabled. See
+  documentation of the flag for details.
+  (`#10390 <https://github.com/coq/coq/pull/10390>`_,
+  by Gaëtan Gilbert).
+- **Added:**
+  Built-in support for persistent arrays, which expose a functional
+  interface but are implemented using an imperative data structure, for
+  better performance.
+  (`#11604 <https://github.com/coq/coq/pull/11604>`_,
+  by Maxime Dénès and Benjamin Grégoire, with help from Gaëtan Gilbert).
+- **Fixed:**
+  A loss of definitional equality for declarations obtained through
+  :cmd:`Include` when entering the scope of a :cmd:`Module` or
+  :cmd:`Module Type` was causing :cmd:`Search` not to see the included
+  declarations
+  (`#12537 <https://github.com/coq/coq/pull/12537>`_, fixes `#12525
+  <https://github.com/coq/coq/pull/12525>`_ and `#12647
+  <https://github.com/coq/coq/pull/12647>`_, by Hugo Herbelin).
+- **Changed:** Primitive arrays are now irrelevant in their single
+  polymorphic universe (same as a polymorphic cumulative list
+  inductive would be) (`#13356
+  <https://github.com/coq/coq/pull/13356>`_, fixes `#13354
+  <https://github.com/coq/coq/issues/13354>`_, by Gaëtan Gilbert).
```

Comment:
I think that this changelog entry should be merged with the one above on support for primitive arrays. In any case, the use of **Changed** in this case is made inappropriate by the fact that it doesn't change a feature that existed in the previous version.

Reply:
Right